insights/parsers/netstat.py

```diff
@@ -677,7 +677,8 @@ class SsTULPN(CommandParser):
     def parse_content(self, content):
         # Use headings without spaces and colons
         SSTULPN_TABLE_HEADER = ["Netid  State  Recv-Q  Send-Q  Local-Address-Port Peer-Address-Port  Process"]
-        self.data = parse_delimited_table(SSTULPN_TABLE_HEADER + content[1:])
+        content = [line for line in content if (('UNCONN' in line) or ('LISTEN' in line))]
+        self.data = parse_delimited_table(SSTULPN_TABLE_HEADER + content)
 
     def get_service(self, service):
```

Comment:
There needs to be a docstring for `get_service` so that it will be included in the documentation.

Reply:
Done

Reply:
@vishwanathjadhav did you check in the docstrings, I don't see them?